CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.42
+
+* Add support for `solo` test and group.
```

Comment:
Be more clear that this is a new param on test and group. And explain a bit of what it does?

Reply:
Done.